model/src/main/kotlin/licenses/LicenseConfiguration.kt

```diff
@@ -84,9 +94,23 @@ data class LicenseConfiguration(
         result
     }
 
-    @Suppress("UNUSED") // This is intended to be mostly used via scripting.
+    /**
+     * Return a set with the licenses that are assigned to the category with the given [name][categoryName].
+     * If the there is no category with the name provided, throw an [IllegalStateException].
+     * This is intended to be mostly used via scripting.
+     */
     fun getLicensesForCategory(categoryName: String): Set<License> =
         licensesByCategoryName[categoryName] ?: error("Unknown license category name: $categoryName.")
+
+    /** A property for fast-lookups of licenses by their ID. */
+    private val licensesById: Map<SpdxSingleLicenseExpression, License> by lazy {
+        categorizations.associateBy { it.id }
+    }
+
+    /**
+     * Returns the [License] for the given [id] or *null* if no such license can be found.
```

Comment:
Maybe we should emphasize that this is just "A convenience operator to return ..."